test/common_test/collisions.cc

```diff
@@ -141,6 +147,130 @@ TYPED_TEST(CollisionTest, MeshAndPlane)
   }
 }
 
+struct CollisionStaticFeaturesList : gz::physics::FeatureList<
+  gz::physics::sdf::ConstructSdfModel,
+  gz::physics::sdf::ConstructSdfWorld,
+  gz::physics::GetContactsFromLastStepFeature,
+  gz::physics::ForwardStep
+> { };
+
+template <class T>
+class CollisionStaticTest :
+  public CollisionTest<T>{};
+using CollisionStaticTestTypes =
+  ::testing::Types<CollisionStaticFeaturesList>;
+TYPED_TEST_SUITE(CollisionStaticTest,
+                 CollisionStaticTestTypes);
+
+TYPED_TEST(CollisionStaticTest, StaticCollisions)
```

Comment:
I started simplifying these tests in https://github.com/gazebosim/gz-physics/pull/493 but never finished. We can just use a `TEST_F` here instead of a `TYPED_TEST`.